### ISSUE TITLE
Modal popup in datatables not working on page 2+ in IE

### ISSUE THREAD

**Issue by locutux** [wet-boew/wet-boew#8973]
I have an modal popup that I create on each line in my table. When I click on the button in the line, it opens the overlay. It works on items in the first page of the table, but right after i switch to page 2, no button works (for any line). Switch back to 1, and it works.

I can see no difference in the inspected source code.

I read multiple tickets here regarding interaction between Datatables and Overlay, and even though the solutions expressed in those issues (like https://github.com/wet-boew/wet-boew/issues/7930) could be a start, our data transfer methods are different than those given. Do you have any other solution we could try on our end, please?

Thank you for your time and have a great day!

**Comment by thekodester** [comment]
Yes I talked about this issue before, you can see a working demo here thekodester.ca/wet/demos/tables/overlay

The reason is related to DataTables (3rd party plugin). See:

Q. My events don't work on the second page
A. When attaching events to cells in a table controlled by DataTables, you need to be careful how it is done. Because DataTables removes nodes from the DOM, events applied with a static event listener might not be able to bind themselves to all nodes in the table. To overcome this, simply use jQuery delegated event listener options, as shown in this example. Additionally, you could use my Visual Event bookmarklet to help debug event issues.

Found at https://datatables.net/faqs/#events.
